src/BaseBlockTracker.ts

```diff
@@ -24,6 +27,7 @@ export abstract class BaseBlockTracker extends SafeEventEmitter {
 
     // config
     this._blockResetDuration = opts.blockResetDuration || 20 * sec;
+    this._usePastBlocks = Boolean(opts.usePastBlocks);
```

Comment:
I recommend adding a default value for this, to allow for backwards compatibility.